Applications/WebPortal/WebApp/library/source/pages/group_page/group_info_page/group_info_page.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { AccountEntry, DisplaySize, PageWrapper } from '../../..';
+import { GroupMemberEntry } from './group_member_entry';
+
+interface Properties {
+
+  /** The size of the viewport. */
+  displaySize: DisplaySize;
+
+  /** The accounts that belong to the group. */
+  group: AccountEntry[];
+}
+
+/** A page that displays information about a group and its members. */
+export class GroupInfoPage extends React.Component<Properties> {
+
+  public render(): JSX.Element {
+    const {displaySize, group} = this.props;
+    const membersCount = (() => {
+      if(displaySize !== DisplaySize.SMALL) {
+        return (
+          <span style={GroupInfoPage.STYLE.membersInfo}>
+            Total members: {group.length}
+          </span>);
+      } else {
+        return null;
+      }
+    })();
+    const pageWidth = (() => {
+      if(displaySize === DisplaySize.SMALL) {
```

Comment:
Experiment with something like:

    const pageWidth = GroupInfoPage.STYLE[this.props.displaySize];

You'll then need to define it along the lines of:

    private static readonly STYLE = {
      [DisplaySize.SMALL]: { ... },
      [DisplaySize.MEDIUM]: { ... }
    };

You might need to make some modifications to that syntax but the idea overall should work.

Reply:
I didn't realize that would work! Going forward I'll  do that more.